### PR TITLE
Fix: Return emails when fetching User from DB

### PR DESCRIPTION
**Pull request by ryanahamilton**
## Description

In Prisma2, the `emails` field is not part of the default selection set. This `include` explicitly tells it to return `emails` when querying Users.

## 🎟 Issue(s)

Resolves un-filed issue identified by @kushalmalani:

> Unable to see email address when selecting a workspace user on dev.  Works fine on staging though. This is causing for updateUser call to fail with the following in console
>
> `Uncaught TypeError: user.emails is null`
> image

## 🧪 Functional Testing

Open to suggestions on how we might extend our tests to better protect against this?

## 📸 Screenshots

Response now includes values for `emails` field and is displayed in the UI:

Image 2020-09-18 at 10 20 12 AM

## 📋 Checklist

- [x] Rebased from the main (or release if patching) branch (before testing)
- [x] Ran `yarn lint && yarn test` before taking out of draft
- [x] Added/updated applicable tests
- [x] Tested against [Astro-UI](https://github.com/astronomer/astro-ui/) and [Astro-CLI](https://github.com/astronomer/astro-cli/) clients.
- [x] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [x] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: src/resolvers/query/users/index.js [hunk 1]
```diff
@@ -12,5 +12,8 @@ export default async function users(parent, args, ctx) {
   const query = userQuery(args);
 
   // Run final query
-  return ctx.prisma.user.findMany({ where: { ...query } });
+  return ctx.prisma.user.findMany({
```
Comment (adam2k) [inline src/resolvers/query/users/index.js:15]: Is this paginated or will this return every user?

File: src/resolvers/query/users/index.js [hunk 2]
```diff
@@ -12,5 +12,8 @@ export default async function users(parent, args, ctx) {
...
+    where: { ...query },
+    include: { emails: true }
```
Comment (adam2k) [inline src/resolvers/query/users/index.js:17]: Is the `email` the only field we need here or should we include other values like `fullName`, `profile`, etc?

**Review by adam2k — APPROVED** [review]
This looks good to me.  I have a couple questions about the query that I left, but they might not need to be addressed right away.

**Review by kushalmalani — APPROVED** [review]

File: src/resolvers/query/users/index.js (hunk 2 above)
Comment (andriisoldatenko) [reply src/resolvers/query/users/index.js:17]: AFAIK it will return all fields except related fields.

**Review by andriisoldatenko — APPROVED** [review]

**Pull request merged**